frontend/src/styles/themes/variables.css

```
@@ -64,7 +64,7 @@
   --color-border-dark: #374151;
 
   /* Typography */
-  --font-family-sans: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  --font-family-sans: 'Segoe UI', Tahoma, Geneva, Verdana, 'Microsoft YaHei', 'PingFang SC', 'Noto Sans CJK SC', sans-serif;
```

Stylelint (17.6.0)

[error] Expected "Tahoma" to be "tahoma" (value-keyword-case)

(value-keyword-case)